apps/desktop-wallet/src/api/apiDataHooks/address/useFetchAddressFts.ts

```diff
@@ -18,34 +18,32 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import { AddressHash } from '@alephium/shared'
 
-import useAddressAlphBalances from '@/api/apiDataHooks/address/useAddressAlphBalances'
-import useAddressTokensByType from '@/api/apiDataHooks/address/useAddressTokensByType'
-import useSortFTs from '@/api/apiDataHooks/useSortFTs'
+import useFetchAddressTokensByType from '@/api/apiDataHooks/address/useFetchAddressTokensByType'
+import { SkipProp } from '@/api/apiDataHooks/types'
+import useFetchSortedFts from '@/api/apiDataHooks/utils/useFetchSortedFts'
 
-interface UseAddressFTsProps {
+interface UseAddressFTsProps extends SkipProp {
   addressHash: AddressHash
   sort?: boolean
 }
 
-const useAddressFTs = ({ addressHash, sort = true }: UseAddressFTsProps) => {
-  const { data: alphBalances, isLoading: isLoadingAlphBalances } = useAddressAlphBalances({ addressHash })
+const useFetchAddressFts = ({ addressHash, sort = true, skip }: UseAddressFTsProps) => {
   const {
-    data: { listedFTs, unlistedFTIds },
+    data: { listedFts, unlistedFTIds },
```

Comment:
Oops, I see that `unlistedFTIds` was not renamed to `unlistedFtIds`, I'll take care of it in my next PR together with other renames to avoid making this PR even larger than it is.